src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -502,6 +529,31 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
     return representations
   }
 
+  buildRepresentationsZipBothBodyshape(contents: SortedContent): WearableRepresentation[] {
```

Comment:
This method builds the representations from a .zip file containing both representations.